lib/tasks/schedule_rules.rake

```diff
@@ -23,4 +23,20 @@ namespace :schedule_rule do
       end
     end
   end
+
+  desc "Adds missing global price groups to existing schedule rules"
+  task add_missing_price_group_discounts: :environment do |_t, _args|
+    ScheduleRule.all.find_each do |schedule_rule|
```

Comment:
This is similar to `PriceGroup#setup_schedule_rules`. That could be used, but we need to get the current `percent_discount` rather than setting it to 0